main.py

```diff
@@ -1,10 +1,20 @@
-import cv2
+import cv2, face_recognition
 #get webcam
 cap = cv2.VideoCapture(0)
 while True:
 	q , frame = cap.read()
+	#detect the faces
+	for top,right,bottom,left in face_recognition.face_locations(frame):
+		
+		#draw rectangle around faces
```

Comment:
Comments around each line can pad the file too much.
Use comments to explain "why" you did something if it doesn't feel obvious.
The line below this is clear enough to not need a comment.